### ISSUE TITLE
Syntax Error when establishing a WebSocket

### ISSUE THREAD

**Issue by Congee** [Kozea/wdb#118]
Hey dude, this piece of code should be

``` diff
     function Websocket(wdb, uuid) {
       var proto;
       this.wdb = wdb;
       Websocket.__super__.constructor.apply(this, arguments);
       proto = document.location.protocol === "https:" ? "wss:" : "ws:";
+++    this.url = `${proto}//` + document.location.host + "/websocket/" + uuid;
---    this.url = "{proto}//" + document.location.host + "/websocket/" + uuid;
       this.log('Opening new socket', this.url);
       this.ws = new WebSocket(this.url);
       this.ws.onclose = this.close.bind(this);
       this.ws.onopen = this.open.bind(this);
       this.ws.onerror = this.error.bind(this);
       this.ws.onmessage = this.message.bind(this);
     }

```
https://github.com/Kozea/wdb/blob/master/server/wdb_server/static/javascripts/wdb.js#L73

Also, the pypi package is obsolete. No static/hipster.jpg. web.server.py --unminified is broken because wdb.js is not bundled in.

**Comment by yelizariev** [comment]
:+1: 
This should be related to the issue I got:

`deps.min.js?v=1e246b3712a948256db33348518103be:3 Uncaught DOMException: Failed to construct 'WebSocket': The URL '{proto}//localhost:1984/websocket/0fada9b6-4e98-449f-8ad1-e1e222502b25' is invalid.`

**Comment by tsabi** [comment]
Confirm, without this the debugging console don't work!

**Comment by paradoxxxzero** [comment]
Sorry I don't have access to a computer right now, I'll look into it asap

Le dim. 10 juin 2018 à 19:36, Csaba Tóth <notifications@github.com> a
écrit :

**Comment by grafa** [comment]
+1
this broke my dev env

**Comment by paradoxxxzero** [comment]
Sorry about that and that should have been fixed with the 3.2.4 version

Le ven. 15 juin 2018 à 13:46, Pavel Hrdina <notifications@github.com> a
écrit :

**Comment by grafa** [comment]
Just upgraded to 3.2.4 and it's fixed :tada:

**Comment by Congee** [comment]
Fixed by  d222596
